python/pyspark/pandas/generic.py

```diff
@@ -946,6 +947,7 @@ def to_json(
             the options in PySpark's API documentation for `spark.write.json(...)`.
             It has a higher priority and overwrites all other options.
             This parameter only works when `path` is specified.
+        ignoreNullFields: if set to True and path is provided, writer omits columns with all NaN or Null values.
```

Comment:
I think it would be even better if we can just convert this to something like:


```
.. note:: Set ignoreNullFields keyword argument to `True` to blah blah .. 
```